app/default/data.py

```diff
@@ -178,8 +179,12 @@ def get_fund_data(fund_id, language=None, as_dict=False):
 
 
 def get_fund_data_by_short_name(fund_short_name, as_dict=False):
+    all_funds = [fund["short_name"].lower() for fund in get_all_funds()]
+    if fund_short_name.lower() not in all_funds:
+        current_app.logger.warning(f"Invalid fund {fund_short_name}!")
```

Comment:
Should this be `fund_short_name.lower()`?

Reply:
It's meant to still be human readable, and should probably match with the input it's referencing. Not sure, I feel it should be the same, but I'm not precious about it.

Reply:
Yeah makes sense I agree